clang/lib/Driver/Driver.cpp

```diff
@@ -7845,7 +7881,7 @@ void Driver::BuildActions(Compilation &C, DerivedArgList &Args,
     if (!UseNewOffloadingDriver)
       OffloadBuilder->makeHostLinkAction(LinkerInputs);
     types::ID LinkType(types::TY_Image);
-    if (Args.hasArg(options::OPT_fsycl_link_EQ))
+    if (Args.hasArgNoClaim(options::OPT_fsycl_link_EQ))
```

Comment:
Why has these been updated to `hasArgNoClaim`?

Reply:
This change was for a different solution I was working for the `-fsycl-link` cleanup.  This isn't necessary anymore - I will update.